en/12_Project_Governance/05_Release_policy.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Release policy
+summary: Outline the release lifecycle and support commitment for Silverstripe CMS
+icon: code-branch
+---
+
+# Silverstripe CMS release policy
+
+This document outlines the unified release policy for Silverstripe CMS and its commercially supported modules. Its purpose is to provide clear guidance on the types of releases, their cadence, scope, and support lifecycle, ensuring predictability and stability for our users and developers.
+
```

Comment:
```suggestion

> [!NOTE]
> Refer to our [definition of public API](/project_governance/public_api/).

```
This is in both existing policies.

Reply:
Updated

Reply:
Not added?

Reply:
Hadn't pushed, updated. It's in the same place as the major/minor policies i.e. below "Community modules are not covered by this policy ..."